### PR TITLE
[20.01] Remove GenomeSpace tools, OpenID, UI and requirements

### PR DESCRIPTION
**Pull request by nsoranzo**
Also:
- backport https://github.com/galaxyproject/galaxy/pull/9357
- update `bx-python`, `galaxy-sequence-utils` and `pycryptodome`

### CODE REVIEW

**Review by nuwang — APPROVED** [review]
LGTM

**Review by mvdbeek — APPROVED** [review]

File: lib/galaxy/dependencies/pipfiles/default/pinned-requirements.txt [hunk 1]
```diff
@@ -28,7 +28,7 @@ boltons==19.3.0
 boto3==1.9.114
 boto==2.49.0
 botocore==1.12.253
-bx-python==0.8.6
+bx-python==0.8.7
```
Comment (nsoranzo) [inline lib/galaxy/dependencies/pipfiles/default/pinned-requirements.txt]: @mvdbeek Should I revert this for the time being given https://github.com/bxlab/bx-python/pull/58 ?

Comment (mvdbeek) [reply lib/galaxy/dependencies/pipfiles/default/pinned-requirements.txt]: Yeah, I'll take care of this in the converter PR (which also tests bx-python).

**Comment by nsoranzo** [comment]
@mvdbeek Merge?

**Pull request merged**